src/models/TunewsElementModel.js

```diff
@@ -0,0 +1,48 @@
+// @flow
```

Comment:
@amusameh In file name, the letter `n` of `news` should be capitalized, so it's consistent with all other files referencing Tu News.